### PR TITLE
feat: Added task card into taskRunsPage 

### LINKED ISSUE #1495: Task Runs Page UX Touch Up
### Background
The tasks runs page could be made more usable with very little effort. 

A screenshot from [the figma mockup](https://www.figma.com/file/RzQhMu3r9tyY9IYEl0JpmK/Untitled?node-id=0%3A1)

#### Main interface

*Essentially: replicate the task card on the tasks list page*

#### Breadcrumbs

*We don't have a lot of breadcrumb type navigation in the app - this might require some iteration*

### Acceptance Criteria
- The page includes a way to navigate back to the tasks list page
- The page shows the status (`active` | `inactive`) of the task
- The page has a toggle to change the status of the task
- Re-running a task should re-load the page
- UX tweaks (e.g. spacing between `View Logs` and `Retry`)

### PR DESCRIPTION
**Pull request by blegesse-git**
See: #1495

To make code review/ merging easier, I've split #1655 into smaller portions. 
This PR contains some small portion of changes implemented by PR #1655. 

Added breadcrumb navigation that takes user from task runs page to task page. 
replicated task card into task runs page.
fixed spacing issue between 'view logs' and 'retry' button in task list row page 

Screen Shot 2021-06-15 at 1 29 46 PM

### CODE REVIEW

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 56 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=1717)

**Comment by Asalem1** [comment]
Should #1655 be closed?

File: src/tasks/components/TaskHeaderCard.tsx [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+    )
+  }
+
+  private handleRunTask = () => {
+    const {onRunTask, match, getRuns} = this.props
+    try {
+      onRunTask(match.params.id)
+      getRuns(match.params.id)
+    } catch (error) {
+      console.error(error)
```
Comment (hoorayimhelping) [inline src/tasks/components/TaskHeaderCard.tsx:108]: let's remove this `console.error` and instead dispatch a notification to the user

File: src/tasks/components/TaskHeaderCard.tsx [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+const mdtp = {
+  onAddTaskLabel: addTaskLabel,
+  onDeleteTaskLabel: deleteTaskLabel,
+  onRunTask: runTask,
+  getRuns,
+}
+
+const connector = connect(mstp, mdtp)
+
+export default connector(withRouter(TaskHeaderCard))
```
Comment (hoorayimhelping) [inline src/tasks/components/TaskHeaderCard.tsx:162]: like we mentioned in Zoom, change to named export. [Example here](https://github.com/influxdata/ui/blob/master/src/templates/containers/CommunityTemplatesIndex.tsx#L300)

File: src/tasks/components/PageBreadcrumbs.tsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+          <div key={page.text} className="pointer">
+            {getCrumb(page, isItTheLastPage)}
+          </div>
+        )
+      })}
+    </FlexBox>
+  )
+}
+
+export default PageBreadcrumbs
```
Comment (hoorayimhelping) [inline src/tasks/components/PageBreadcrumbs.tsx:45]: change this to a named export please

File: src/tasks/components/TaskHeaderCard.tsx [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+interface PassedProps {
+  task: Task
+  onActivate: (task: Task) => void
+  onEditTask: () => void
+}
+
+type ReduxProps = ConnectedProps<typeof connector>
+type Props = PassedProps & ReduxProps
+
+export class TaskHeaderCard extends PureComponent<
```
Comment (hoorayimhelping) [inline src/tasks/components/TaskHeaderCard.tsx:41]: maybe rename this to indicate it's used on the tasks runs page.

File: src/tasks/components/TaskHeaderCard.tsx [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+  public render() {
+    const {task} = this.props
+
+    if (!task) {
+      return null
+    }
+
+    return (
+      <ResourceCard
+        testID="task-card"
```
Comment (hoorayimhelping) [inline src/tasks/components/TaskHeaderCard.tsx:53]: you probably want to scope this a little tighter, something like

```
"task-runs-task-card"
```

to provide semantic meaning that it's a task card on the task runs page

File: src/tasks/components/PageBreadcrumbs.tsx [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+const PageBreadcrumbs: FC<Props> = ({pages}) => {
+  const getCrumb = (page: PageInfo, isItTheLastPage: boolean) => {
+    const text = `${page.text}${!isItTheLastPage ? ' >' : ''}`
+
+    if (!page.onClick) {
+      return <h1 className="bread-crumb-title">{text}</h1>
+    } else {
+      return (
+        <h1 id="bread-crumb-title" onClick={page.onClick}>
```
Comment (hoorayimhelping) [inline src/tasks/components/PageBreadcrumbs.tsx:23]: based on the css chat, let's change this to a class.

File: src/tasks/components/TaskHeaderCard.tsx (hunk 1 above)
Comment (blegesse-git) [reply src/tasks/components/TaskHeaderCard.tsx]: while i was working on implementing notify, i realized there is a function in handleRunTask that handles the error by notifying the user. 
so there is a code that does exactly what we want it to do. 
Screen Shot 2021-06-16 at 11 22 36 AM

**Review by hoorayimhelping — APPROVED** [review]
great work! thanks for splitting this up

**Pull request merged**